.github/ISSUE_TEMPLATE/feature-request--new-addition-to-addons-.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request (new addition to addons)
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Can we refer to https://github.com/tensorflow/tensorflow/blob/master/.github/ISSUE_TEMPLATE/30-feature-request.md ?

By the way, the file name looks unnatural, do you mean `feature-request.md`?

Reply:
Agree. As part of the upcoming RFC on maintainability could we also add a bullet to that which says 
> Willing to maintain?

Reply:
I agree. As part of the upcoming RFC could we also add
> Willing to maintain?

Reply:
added :)